src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Services/SchemaService.cs

```diff
@@ -27,6 +27,7 @@ public SchemaService(
     public async Task<ISchema> GetSchemaAsync()
     {
         var hasChanged = false;
+        var culture = CultureInfo.CurrentUICulture;
```

Comment:
I thought about that too, but IMHO this doesn't buy us anything. `CultureInfo.CurrentUICulture` is ultimately used by the `IStringLocalizer` to which we must align the schema anyway. So if `IRequestCultureFeature` ever returns something else (which it shouldn't) we would end up with the wrong translations. 
As it normally returns the same value, it only adds one more indirection and wastes CPU cycles.

Reply:
Then why is `IRequestCultureFeature` needed? It's seems to be used by many services including the DefaultContentLocalizationManager.

Reply:
As far as I know, it allows to retrieve information about which provider has set the current culture. This is not needed at this point.

Reply:
We have a middleware (the aspnet one) that uses all the culture providers and sets the `CultureInfo.CurrentUICulture`, so it's fine to use it.

But then, where you generate the link with the `?culture={CultureInfo.CurrentUICulture}` I don't understand why you need that. Since the next request will still use the active culture, no need to force it. (but you may also educate me)

Reply:
> But then, where you generate the link with the `?culture={CultureInfo.CurrentUICulture}` I don't understand why you need that. Since the next request will still use the active culture, no need to force it. (but you may also educate me)

This forces the culture of the schema retrieved by GraphiQL to be the same as the selected admin culture. It doesn't work automatically, because the `AdminCookieCultureProvider`, which is used to set the current culture for the admin dashboard, checks for the `admin` path prefix. The `api/graphql` path is therefore not picking up the admin culture cookie.